documentation-site/examples/tree-view/overrides.js

```diff
@@ -83,10 +83,10 @@ export default function TreeViewOverrides() {
           },
         },
         CollapseIcon: {
-          component: CheckIndeterminateIcon,
+          component: props => <CheckIndeterminateIcon {...props} />,
```

Comment:
This signals a huge problem with the diff. If an icon is passed to overrides it will fail flow check.

Reply:
wait, how is this not a breaking change?

Reply:
Yeah that's my point. Although, the previous code should work the same it will just not pass flow-check

Reply:
sigh.. you're right these two things are not equivalent